cogs5e/initiative/cog.py

```diff
@@ -227,9 +252,24 @@ async def add(self, ctx, modifier: int, name: str, *args):
             init_roll_skeleton = str(init)
 
         me = Combatant.new(
-            name, controller, init, init_skill, hp, ac, private, Resistances.from_dict(resists), ctx, combat
+            name, controller, init, init_skill, hp, ac, private, Resistances.from_dict(resists), ctx, combat, stats
```

Comment:
I'd reorganize the order of these args so `ctx` and `combat` aren't kind of floating in the middle there - I can handle that after the other changes